core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -135,31 +124,6 @@ public HttpResponse doCompleteInstall(StaplerRequest req, StaplerResponse rsp) t
         j.setSetupWizard(null);
         return HttpResponses.okJSON();
     }
-
-    // Stores a user property for the authentication key, which is really the auto-generated user's password
-    public static class AuthenticationKey extends UserProperty {
```

Comment:
This code has been injected in 2.0, so it's fine. BTW missing <code>@since</code> causes some extra review work